design_app/src/pages/Wire-backup.jsx

```diff
@@ -0,0 +1,656 @@
+import React from 'react';
```

Comment:
Is this file still required?

Reply:
No, not really. I cretaed it just in case Darko did some changes. I'm removing it.

Reply:
Removed.